### ISSUE TITLE
SVGLandmarkRepresentation does not respect sized viewports

### ISSUE THREAD

**Issue by llangholz89** [Kitware/vtk-js#1754]
I'm not sure if this is already being covered in the other work going on around widgets, but I've noticed that the SVGLandmarkRepresentation (and likely all other SVG widgets) is not positioned properly when working with renderers that are not full-sized (i.e. `renderer.setViewport([0.1, 0.2, 0.9, 08.])`)

To reproduce, simply add a line in the `LineWidget` example to size the viewport:
```
...
const fullScreenRenderer = vtkFullScreenRenderWindow.newInstance({
  background: [0, 0, 0],
});
const renderer = fullScreenRenderer.getRenderer();
renderer.setViewport([0.1, 0.2, 0.9, 0.8]); // <-- set a specific size on the viewport
...
```

I looked into it only a little bit, but got stuck on how to properly calculate the appropriate "y" coordinate for the SVG, and started wondering if more info was needed from the PixelSpaceCallbackMapper's callback. But I'm not super familiar with that mechanism so I figured I'd post this here before going down that rabbit hole...

![ezgif com-gif-maker](https://user-images.githubusercontent.com/16705837/109104607-224b2000-76e1-11eb-8c82-81c29e2bfe10.gif)

**Comment by jourdain** [comment]
I think @finetjul fix that problem but I might be wrong.

**Comment by finetjul** [comment]
Nope, this has not been fixed. This is unrelated to the recent fixes regarding HighDPI. Should be related to  "world to display coordinate" conversion.
